WordPress/Classes/ViewRelated/Themes/ThemeBrowserFactory.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+class ThemeBrowserFactory : NSObject {
```

Comment:
Factories are a bit uncommon in the iOS realm. Main reason is that it's an extra file / extra class name (if you check, this one would be the first *Factory class in the project.

Common way of doing this is by simply adding a static method... `static func browserWithBlog(blog: Blog) -> ThemeBrowserViewController`, to the browser class instead, which would contain just 4 lines of code, instead of an entire new class.

Absolutely no need to change anything, this is just a side comment!.